src/RSocketServer.cpp

```diff
@@ -119,7 +119,9 @@ void RSocketServer::onRSocketResume(
   // We don't need to check for isShutdown_ here since all callbacks are
   // processed by this time.
 
-  CHECK(false) << "not implemented";
+  LOG(ERROR) << "onRSocketResume: not implemented";
+  frameTransport->close(
```

Comment:
Need to name the function's parameter, it got purged in a previous diff where I removed all unused parameters.